microcosm_logging/factories.py

```diff
@@ -113,20 +112,17 @@ def make_json_formatter(graph):
     Used for searchable aggregation.
 
     """
-    fields = {
-        # what was logged?
+    fields = """{
         "loggerName": "%(name)s",
         "levelName": "%(levelname)s",
-        "message": "%(message)s",
```

Comment:
@sethisernhagen  if you look at the code for the actual `logging.formatter` class here: http://bugs.python.org/file4410/logging.py you should see how to subclass it. The underlying formatter has more control over the `record` used as input than we have here and should allow for a cleaner solution.